lib/ansible/plugins/action/__init__.py

```diff
@@ -869,6 +870,10 @@ def _low_level_execute_command(self, cmd, sudoable=True, in_data=None, executabl
             display.debug("_low_level_execute_command(): using become for this command")
             cmd = self._play_context.make_become_cmd(cmd, executable=executable)
 
+        if chdir:
+            display.debug("_low_level_execute_command(): changing cwd to %s for this command" % chdir)
+            cmd = self._connection._shell.append_command('cd %s' % chdir, cmd)
```

Comment:
Thank you for your review.
It made me realize that the `cd` should also be sudoed.